.github/workflows/Deploy-dev.yml

```diff
@@ -0,0 +1,18 @@
+name: Deploy-dev
+
+on:
+  push:
+    branches-ignore:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: akhileshns/heroku-deploy@v3.0.4 # This is the action
+        with:
+          heroku_api_key: ${{secrets.HEROKU_API_KEY}}
+          heroku_app_name: "bonspiel-tracking-system-devl" #Must be unique in Heroku
```

Comment:
Can we change this from `bonspiel-tracking-system-devl` to `bonspiel-tracking-system-dev` since the workflow is called Deploy-dev?

Reply:
That name wasn't available!

Reply:
Lmao that's lame.